internal/cephfs/core/volume.go

```diff
@@ -191,6 +191,23 @@ func CreateVolume(ctx context.Context, volOptions *VolumeOptions, volID fsutil.V
 	return nil
 }
 
+// ExpandVolume will expand the volume if the requested size if greater than
```

Comment:
if the requested size if greater than  -> if the requested size "is" greater than